apps/src/lib/node/ledger/tendermint.rs

```diff
@@ -505,3 +509,28 @@ fn write_validator_key(
     });
     file.write(key.to_string().as_bytes()).map(|_| ())
 }
+
+#[cfg(feature = "dev")]
+fn write_chain_id(
+    home_dir: impl AsRef<Path>,
+    chain_id: String,
+) -> io::Result<()> {
+    let home_dir = home_dir.as_ref();
+    let path = home_dir.join("config").join("genesis.json");
+
+    // Don't use `TendermintConfig::load_genesis_file()` because tendermint
+    // 0.34.x requires `block.TimeIotaMs`. It doesn't exist in tendermint-rs.
+    // In the master of Tendermint (go), it was also removed at
+    // [#5987](https://github.com/tendermint/tendermint/pull/5987))
```

Comment:
It looks like some other fields are outdated too, but `block.TimeIotaMs` has not been released in Tendermint(go) 0.34.x. I will probably be in 0.35. I'll open an issue for this

Reply:
I've pushed a commit to use an updated branch. The other changes we previously upstreamed to tendermint-rs have been released so I made a new branch from their master.

Reply:
Thank you! That's pretty cool.